rust/main/agents/relayer/src/msg/pending_message.rs

```diff
@@ -888,22 +895,23 @@ mod test {
     fn test_calculate_msg_backoff_does_not_overflow() {
```

Comment:
why did this test not fail before? I don't understand why it was overflowing as well

Reply:
discussed irl, it was overflowing because the backoff period was added to `Instant::now()` inside `maybe_from_persisted_retries`. The unit test only checked that the call to `calculate_msg_backoff` didn't overflow, which I hadn't realised was used in an addition afterwards